scripts/us_census/pep/monthly_population_estimate/download.py

```diff
@@ -0,0 +1,220 @@
+        df = _clean_txt_file(df, SCALING_FACTOR_TXT_FILE)
+        df.to_excel(download_local_path + os.sep + file_name,
+                    index=False,
+                    engine='xlsxwriter')
+
+
+def _sum_cols(col: pd.Series) -> pd.Series:
+    """
+    This method concats two DataFrame column values
+    with space in-between.
+
+    Arguments:
+        col[0] (Series) : DataFrame Column of dtype str
+        col[1] (Series) : DataFrame Column of dtype str
+
+    Returns:
+        res (Series) : Concatenated DataFrame Columns
+    """
+    res = col[0]
+    if col[1] is None:
+        return res
+    res = col[0] + ' ' + col[1]
+    return res
+
+
+def _mulitply(col: pd.Series, **kwargs: dict) -> pd.Series:
+    """
+    This method multiply dataframe column with scaling factor.
+
+    Arguments:
+        col (Series): DataFrame Column of dtype int
+        **kwargs (dict): Dict with key 'scaling_factor' and value type int
+
+    Returns:
+        res (Series): DataFrame column values mulitplied by scaling_factor.
+    """
+    res = col
+    if col not in [None, np.NAN]:
+        if col.isdigit():
+            res = int(col) * kwargs["scaling_factor"]
+    return res
+
+
+def _clean_csv_file(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    This method cleans the dataframe loaded from a csv file format.
+    Also, Performs transformations on the data.
+
+    Arguments:
+        df (DataFrame) : DataFrame of csv dataset
+
+    Returns:
+        df (DataFrame) : Transformed DataFrame for txt dataset.
+    """
+    idx = df[df[0] == "Year and Month"].index
+    df = df.iloc[idx.values[0] + 1:][:]
+    df = df.dropna(axis=1, how='all')
+    cols = [
+        "Year and Month", "Resident Population",
+        "Resident Population Plus Armed Forces Overseas", "Civilian Population",
+        "Civilian NonInstitutionalized Population"
+    ]
+    df.columns = cols
+    for col in df.columns:
+        df[col] = df[col].str.replace(",", "")
+    return df
+
+
+def _clean_txt_file(df: pd.DataFrame,
+                    scaling_factor_txt_file: int) -> pd.DataFrame:
+    """
+    This method cleans the dataframe loaded from a txt file format.
+    Also, Performs transformations on the data.
+
+    Arguments:
+        df (DataFrame) : DataFrame of txt dataset
+        scaling_factor_txt_file (int) : Scaling factor for text file
+
+    Returns:
+        df (DataFrame) : Transformed DataFrame for txt dataset.
+    """
+
+    df['Year and Month'] = df[['Year and Month', 'Date']]\
+                                    .apply(_sum_cols, axis=1)
+    df.drop(columns=['Date'], inplace=True)
+    for col in df.columns:
+        df[col] = df[col].str.replace(",", "")
+    idx = df[df['Resident Population'] == "(census)"].index
+
+    resident_population = 1
+    resident_population_plus_armed_forces_overseas = 2
+    civilian_population = 3
+    civilian_noninstitutionalized_population = 4
+
+    #Moving the row data left upto one index value.
```

Comment:
nit: space between '#' and 'M' ... in all comments

Reply:
comments have been changed as per convention